person.rb

```diff
@@ -1,47 +1,22 @@
 #!/usr/bin/env ruby
-class Animal
-    attr_accessor :name
-    def initialize(type, number_of_legs, name = "Unknown")
-        @id = Random.rand(1..1000)
-        @name = name
-        @number_of_legs = number_of_legs
-        @type = type
-    end
-
-    def speak
-        if @type == "dog"
-            "Woof, Woof"
-        elsif @type == "spider"
-            "..."
-        end
-    end
-
-    def bring_a_stick
-        if @type =="dog"
-            "Here your stick ---------"
-        end
-    end
-
-    def make_a_web
-        if@type == "spider"
-            "www"
-        end
-    end
-
-    def id
-        @id
-    end
-
-    def type 
-        @type
-    end
-
+class Person
+  def initialize(age:, name: 'Unknown', _parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+  end
+
+  attr_accessor :name, :age
+  attr_reader :id
+
+  # I have to disable this linter about Naming in order to stick with the task.
+  # rubocop:disable Naming/PredicateName
+  def is_of_age?
+    @age >= 18
+  end
+
+  # rubocop:enable Naming/PredicateName
+  def can_use_services?
+    is_of_age? || parent_permission
+  end
```

Comment:
- [ ] Don't forget to change the parent permission in line 20 after you change the initialize to save the argument.

```suggestion
  def can_use_services?
    is_of_age? || @parent_permission
  end
```

Reply:
Thank you for the tip